r/tests/testthat/test-RecordBatch.R

```diff
@@ -120,6 +120,23 @@ test_that("head and tail on RecordBatch", {
   expect_identical(as.data.frame(tail(batch, -4)), tail(tbl, -4))
 })
 
+test_that("RecordBatch print method", {
+  expect_output(
+    print(batch),
+    paste(
+      "RecordBatch",
+      "10 rows x 5 columns",
+      "$int <int32>",
+      "$dbl <double>",
+      "$lgl <bool>",
+      "$chr <string>",
+      "$fct <dictionary<values=string, indices=int8, ordered=0>>",
```

Comment:
Just for the record, R doesn't have boolean values? ("ordered=0")

Reply:
It does, but this is coming straight from the C++ `ToString` method. I guess I could munge it in R.

Reply:
I suppose it's fine then.

Reply:
I added some munging in https://github.com/apache/arrow/pull/5492/commits/02afb89f24dccecd88176d483bb406c82908c781#diff-63df045aa8aff63fdd460b816987ba0dR65-R69